webapp.py

```diff
@@ -0,0 +1,79 @@
+from flask import Flask, render_template, request, redirect, url_for
+from main import RESULT_DIR, FOLDER_TYPES, TARGET_FOLDER
+import lib.arrange
+import lib.utils
+import webbrowser
+import time
+import os
+
+app = Flask(__name__)
+destination = os.path.join(TARGET_FOLDER, RESULT_DIR)
+
+
+def startwork():
+    """Create folders to keep files according to their file types"""
+    destination = os.path.join(TARGET_FOLDER, RESULT_DIR)
+    lib.utils.makeFolders(destination, FOLDER_TYPES.keys())

```

Comment:
Can we rather use the original function that was defined in `utils`?

Reply:
Yes, but we will need a different target folder and destination values when we are working on different directories. This will update the values only; the execution code will be imported from utils, no change on that